submissions/DaniaB24/friends-app/js/app.js

```diff
@@ -0,0 +1,103 @@
+const cardsOfFriends = document.getElementById("all_humans");
+const searchBar = document.getElementById("search");
+const filterMenu = document.getElementById("filter_menu");
+
+let friends;
+let filteredUsers;
+function startApp() {
+  const URL = "https://randomuser.me/api?results=10";
+  fetch(URL)
+    .then(handleErrors)
+    .then((response) => response.json())
+    .then((users) => {
+      console.log("ok");
+      friends = users.results;
+      filteredUsers = users.results;
+      displayUsers(friends);
+      searchFriends(filteredUsers);
+      filterMenu.addEventListener("click", (event) => {
+        filterSetting(event, friends);
+      });
+    })
+    .catch((error) => {
+      alert("Please reload, Authorization failed : " + error.message);
+      console.log("Authorization failed : " + error.message);
+    });
+}
+function handleErrors(response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+function filterSetting({ target }, friends) {
+  const sortedResults = sortFriends(target, friends);
+  filterByGender(sortedResults ?? []);
+}
+function displayUsers(friends) {
+  const createCards = friends
+    .map((item) => {
+      return `
+      <div class="card_person">
+         <div class="card_container">
+           <img src="${item.picture.large}">
+           <h3 class="card_h3">${item.name.first} ${item.name.last}</h3>
```

Comment:
`div class='button`' :) You don't need `h3` in the classname for h3

Reply:
lol, sorryyy))))))))))))))